src/Symfony/Bundle/FrameworkBundle/Tests/Kernel/KernelHelperTraitTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Kernel;
+
+use Symfony\Bundle\FrameworkBundle\Kernel\KernelHelperTrait;
+use Symfony\Bundle\FrameworkBundle\Tests\TestCase;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+class KernelHelperTraitTest extends TestCase
+{
+    public function testForward()
+    {
+        $request = Request::create('/');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Symfony\Component\HttpFoundation\Request' in method 'testForward'.](https://www.codacy.com/app/git_9/symfony/file/2332002732/issues/source?bid=3207821&fileBranchId=3207915#l28)